cloudos_cli/__main__.py

```diff
@@ -1225,6 +1225,21 @@ def job_details(ctx,
 @click.option('--archived',
               help=('When this flag is used, only archived jobs list is collected.'),
               is_flag=True)
+@click.option('--filter-status',
+              help='Filter jobs by status (e.g., completed, running, failed, aborted).')
+@click.option('--filter-job-name',
```

Comment:
Here I would add that the search is case insensitive PGTI or pgti will be found, which is in line with how the UI does it